components/sysutils/net-snmp/Makefile

```diff
@@ -91,65 +82,59 @@ PYTHON_ARGS =		--basedir=/usr/bin
 CONFIGURE_OPTIONS  +=	--with-python-modules="$(PYTHON_ARGS)"
 
 # Only build 64 bit Perl support
-PERL_ARGS =	DESTDIR=$(PROTO_DIR) INSTALLDIRS=vendor
-CONFIGURE_OPTIONS.32  +=  PERLPROG="$(PERL.5.22)"
+PERL_ARGS =			DESTDIR=$(PROTO_DIR) INSTALLDIRS=vendor
 CONFIGURE_OPTIONS.32  +=	--disable-embedded-perl
-CONFIGURE_OPTIONS.32  +=  --with-perl-modules="$(PERL_ARGS)"
+CONFIGURE_OPTIONS.32  +=	--without-perl-modules
 CONFIGURE_OPTIONS.64  +=	PERLPROG="$(PERL.5.24)"
 CONFIGURE_OPTIONS.64  +=  --disable-embedded-perl
-CONFIGURE_OPTIONS.64  +=	--with-perl-modules="$(PERL_ARGS)"
+CONFIGURE_OPTIONS.64  +=  --with-perl-modules="$(PERL_ARGS)"
+
+# parallel build not supported
+COMPONENT_BUILD_ARGS=
 
+# Install the net-snmp-57 libraries
 COMPONENT_PRE_CONFIGURE_ACTION = \
-	($(CLONEY) $(SOURCE_DIR) $(@D); $(CLONEY) $(COMPONENT_DIR)/sun $(@D)/sun)
+	($(CLONEY) $(SOURCE_DIR) $(@D); $(CLONEY) $(COMPONENT_DIR)/sun $(@D)/sun; \
+	cd ../net-snmp-57 && $(GMAKE) install);	
```

Comment:
Have to check, why do we need install on pre-configure target

Reply:
I don't know how to let disappear the first 6 commits.
The last one is without TPNO et al.
CONFIGURE_OPTIONS += --sysconfdir=$(ETCDIR)/net-snmp was already added.